configs/log4j.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- $Id$ -->
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/">
+
+  <appender name="STDOUT" class="org.apache.log4j.ConsoleAppender">
+    <param name="File" value="/usr/local/fedora/server/logs/fedoragsearch.log"/>
+    <layout class="org.apache.log4j.PatternLayout">
+      <param name="ConversionPattern" value="%p %d (%c{1}) %m%n"/>
+    </layout>
+  </appender>
+
+  <appender name="FILEOUT" class="org.apache.log4j.FileAppender">
+    <param name="File" value="/usr/local/fedora/server/logs/fedoragsearch.log"/>
+    <param name="append" value="true" />
+    <param name="threshold" value="debug" />
+    <layout class="org.apache.log4j.PatternLayout">
+      <param name="ConversionPattern" value="%p %d (%c{1}) %m%n"/>
+    </layout>
+  </appender>
+
+  <logger name="dk.defxws.fedoragsearch" additivity="false">
+    <level value="INFO" />
+    <appender-ref ref="FILEOUT"/>
+  </logger>
+
+  <logger name="dk.defxws.fgszebra" additivity="false">
+    <level value="INFO" />
+    <appender-ref ref="FILEOUT"/>
+  </logger>
+  
+  <logger name="dk.defxws.fgslucene" additivity="false">
+    <level value="INFO" />
+    <appender-ref ref="FILEOUT"/>
+  </logger>
+
+  <logger name="dk.defxws.fgssolr" additivity="false">
```

Comment:
This look good?
 https://github.com/lutaylor/islandora_vagrant_base_box/commit/65ae56ab067cb164aed53d748218674794d972c7

Gsearch_extensions were updated since I put the main PR in so I had to update the version number. 

Also see https://github.com/Islandora-Labs/islandora_vagrant/pull/128 updated to match.